services/apps/premium/members_enrichment_worker/src/sources/serp/tests.ts

```
@@ -0,0 +1,69 @@
+    //   'website',
+    //   'linkedinFromClearbit',
+    //   'linkedinFromProgai',
+    //   'linkedinFromSerp',
+    // ]
+    // const json2csvParser = new Parser({ fields })
+    // const csv = json2csvParser.parse(members)
+    // writeFileSync('output.csv', csv)
+    // console.log('CSV file has been successfully written.')
+  } catch (err) {
+    console.error('Error writing CSV file:', err)
+  }
```

Biome

[error] This code is unreachable



(lint/correctness/noUnreachable)